sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -513,7 +515,7 @@ export class MessageReceiver extends LinkEntity {
         }
         return;
       } finally {
-        if (this._receiver) {
+        if (this._receiver && !this._stopReceivingMessages) {
           this._receiver.addCredit(1);
         }
```

Comment:
Can this just be `this.addCredit(1);` ?

Reply:
It absolutely can be. Don't know how I missed replacing that one!

Reply:
Fixed.